proposals/lip-0048.md

```diff
@@ -31,36 +31,41 @@ Each chain can configure the token used to pay fees. On the Lisk mainchain, the
 
 ### Minimum Fee per Transaction
 
-As introduced in [LIP 0013][lip-0013], all transactions must have a fee greater or equal to a minimum fee (which can be zero). The minimum fee is computed from the transaction size. Chains can configure their minimum fee per byte. The minimum part of the fee should be burned, this is to avoid that validators can send transactions in the blocks they generate without cost.
+As introduced in [LIP 0013][lip-0013], all transactions must have a fee greater or equal to a minimum fee (which can be zero). The minimum fee is computed from the transaction size. Chains can configure their minimum fee per byte.
+
+The implementation allows the option to transfer minimum fees to the specified pool address `ADDRESS_FEE_POOL`. This could be beneficial for sidechains, as fees collected in this way can be used as an incentive for validators and other protocol participants.
```

Comment:
Right! I understand that "implementation" as a word can be ambiguous and/or problematic, so it's better to avoid it. I like your suggestion and will write something on that line of thought.